models/cms_hcc/cms_hcc_models.yml

```
@@ -95,6 +144,64 @@ models:
       - name: tuva_last_run
         description: The date the model was run.
 
+  - name: cms_hcc__patient_risk_scores_monthly
+    config:
+      schema: |
+        {%- if var('tuva_schema_prefix',None) != None -%}{{var('tuva_schema_prefix')}}_cms_hcc{% else %}cms_hcc{%- endif -%}
+      alias: patient_risk_scores_monthly
+      tags: cms_hcc
+      materialized: table
+    description: >
+      This final model calculates the CMS HCC raw risk score, blended risk score,
+      normalized risk score, and payment risk score for each enrollee in the 
+      payment year and collection period.
+    tests:
+      - unique:
+          column_name: "(patient_id||'-'||payment_year||'-'||collection_end_date)"
```

Use built-in dbt test instead of concat to make it compatible for all supported warehouses.

```suggestion
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - patient_id
            - payment_year
            - collection_end_date
```